I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/gui.rpy", line 66, in script
    define gui.interface_text_font = fonts/BryndanWriteBook-nGPM.ttf
  File "game/gui.rpy", line 66, in <module>
    define gui.interface_text_font = fonts/BryndanWriteBook-nGPM.ttf
NameError: name 'fonts' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/gui.rpy", line 66, in script
    define gui.interface_text_font = fonts/BryndanWriteBook-nGPM.ttf
  File "C:\Users\gtsab\OneDrive\Ambiente de Trabalho\semana interdescuplinar\Nova pasta\renpy-8.3.7-sdk\renpy\ast.py", line 2248, in execute
    self.set()
  File "C:\Users\gtsab\OneDrive\Ambiente de Trabalho\semana interdescuplinar\Nova pasta\renpy-8.3.7-sdk\renpy\ast.py", line 2262, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\gtsab\OneDrive\Ambiente de Trabalho\semana interdescuplinar\Nova pasta\renpy-8.3.7-sdk\renpy\python.py", line 1211, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/gui.rpy", line 66, in <module>
    define gui.interface_text_font = fonts/BryndanWriteBook-nGPM.ttf
NameError: name 'fonts' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.7.25031702
SI 13 1.0
Wed Apr  9 01:26:52 2025
